client/blocks/reader-post-card/style.scss

```diff
@@ -346,10 +338,17 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	display: none;
 }
 
-.reader-post-card__link,
-.reader-post-card__link:visited {
+.reader-post-card__link {
```

Comment:
I thought we could just reuse the link colors being used in post cards, but it was throwing an error and demanded that I namespace it.